File: abc151/2.py
n, k, m = map(int, input().split())

point_list = [int(i) for i in input().split()]
sum_points = sum(point_list)

if m * n - sum_points > k:
    print(-1)
else:
    if m * n - sum_points < 0:
        print(0)
    else:
        print(m * n - sum_points)

File: ts/abc210/2.ts
import * as fs from 'fs';

const input: any = fs.readFileSync('/dev/stdin', 'utf8').split(/ |\n/);

const numbers_of_cards: number = Number(input[0]);
const characters: string = input[1];

// output();
const output = (x: any) => console.log(x)

// takahashi : odd / aoki: even
// see https://gist.github.com/aoi0308/6472388
for (let i: number = 0; i < numbers_of_cards; i++) {
	if (Number(characters[i]) == 1) {
		if (i % 2 == 0) {
			output("Takahashi")
			process.exit(0);
		} else {
			output("Aoki")
			process.exit(0);
		}
	}
  }


